### PR TITLE
Further configuration for api via Stripe Checkout

### PR DESCRIPTION
**Pull request by SeriousMatt**

### CODE REVIEW

**Comment by SeriousMatt** [comment]
Replaced the PHP API with a Node API. The application is now successfully passing tokens including charges, customer data, and card numbers to Stripe's API. To remain PCI compliant, the application is not passing cards manually to Stripe API but is using tokens.

File: stripe-node/app.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+// Set up express
+const app = require("express")();
+const stripe = require("stripe")(keySecret);
+// Todo: Clean up pug form, replace with angular
+app.set("view engine", "pug");
+var bodyParser = require('body-parser')
+// Send header to allow post request
+app.use(bodyParser.json());
+app.use(function(req, res, next) {
+    res.header("Access-Control-Allow-Origin", "*");
```
Comment (mcdurdin) [inline stripe-node/app.js]: Any reason why we are allowing requests from anywhere?

**Review by mcdurdin — APPROVED** [review]

File: stripe-node/app.js (hunk 1 above)
Comment (SeriousMatt) [reply stripe-node/app.js]: Hi Mark, I ran into a blocker restricting requests locally so this is for local development only. I'll add a note to make sure its removed.

**Comment by SeriousMatt** [comment]
These keys have been deactivated.

File: src/app.view.html [hunk 2]
```diff
@@ -2,15 +2,15 @@
...
 <head>
   <meta charset="utf-8">
-  <title><?= $title ?></title>
+  <title>Donate to Keyman</title>
   <base href="/">
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
-  <script src="https://checkout.stripe.com/checkout.js"></script>
+  <script src="https://checkout.stripe.com/checkout.js" data-key="pk_test_g239FhjT812uAnrOdQzUxOrp"></script>
```
Comment (mcdurdin) [inline src/app.view.html:10]: You've still got a key here.

Comment (SeriousMatt) [reply src/app.view.html:10]: Thanks Marc. That key has been deactivated as well.

**Pull request merged**